packages/mui-material/src/FilledInput/filledInputClasses.ts

```diff
@@ -5,8 +5,6 @@ import { inputBaseClasses } from '../InputBase';
 export interface FilledInputClasses {
   /** Styles applied to the root element. */
   root: string;
-  /** Styles applied to the root element if color secondary. */
-  colorSecondary: string;
```

Comment:
Instead of adding the class to the DOM, I chose to remove it here. The fact that no users have reported this issue, even though it has likely existed for a long time (possibly since v5), suggests that it is not widely used. Therefore, I decided to remove the class instead of adding it to the DOM.